### PR TITLE
Chore: add logger and hide secrets

### PR DESCRIPTION
**Pull request by yaronya**
### Feature Request
We shouldn't log secrets.

### Solution
1. Add a `winston` with a custom formatter that masks secrets like: `this-is-top-secret` -> `***************ecret`
2. Replace all `console` usage to use `logger` instead.

### CODE REVIEW

**Comment by yaronya** [comment]
[QA Rejects](https://trello.com/c/0v8P91Pp/1685-qa-rejects)

File: node/src/lib/logger.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+const { createLogger, format, transports } = require('winston');
+const _ = require('lodash');
+const { argumentsMap } = require('../config/arguments');
+
+let secrets = [];
+
+const getSecureSecret = secret => {
```
Comment (avnerenv0) [inline node/src/lib/logger.js:7]: this is a little problematic 
```
> getSecureSecret('abc');
'abc'
> getSecureSecret('12345');
SyntaxError: Invalid regular expression: /^.{1,0}/: numbers out of order in {} quantifier
> getSecureSecret('123');
'123'
> getSecureSecret('123aa');
SyntaxError: Invalid regular expression: /^.{1,0}/: numbers out of order in {} quantifier
```

Comment (avnerenv0) [reply node/src/lib/logger.js:7]: you can just use a static `'******'` here

Comment (yaronya) [reply node/src/lib/logger.js]: 25cf208

**Pull request merged**